File: src/Repository/Utilisateur.php
<?php

namespace App\Repository;

use Doctrine\ORM\Mapping as ORM;

/**
 * Utilisateur
 *
 * @ORM\Table(name="utilisateur")
 * @ORM\Entity
 */
class Utilisateur
{
    /**
     * @var int
     *
     * @ORM\Column(name="ID", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var string|null
     *
     * @ORM\Column(name="NOMCLI", type="string", length=32, nullable=true, options={"fixed"=true})
     */
    private $nomcli;

    /**
     * @var string|null
     *
     * @ORM\Column(name="PRENOMCLI", type="string", length=32, nullable=true, options={"fixed"=true})
     */
    private $prenomcli;

    /**
     * @var string|null
     *
     * @ORM\Column(name="MAILCLI", type="string", length=50, nullable=true, options={"fixed"=true})
     */
    private $mailcli;

    /**
     * @var string|null
     *
     * @ORM\Column(name="PASSCLI", type="string", length=32, nullable=true, options={"fixed"=true})
     */
    private $passcli;

    /**
     * @var string|null
     *
     * @ORM\Column(name="ADRCLI", type="string", length=32, nullable=true, options={"fixed"=true})
     */
    private $adrcli;

    /**
     * @var int|null
     *
     * @ORM\Column(name="CPCLI", type="integer", nullable=true)
     */
    private $cpcli;

    /**
     * @var string|null
     *
     * @ORM\Column(name="VILLECLI", type="string", length=32, nullable=true, options={"fixed"=true})
     */
    private $villecli;


}

File: Kal/Kal/Model/utilisateur.cs
namespace Kal.Model
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("kaldb.utilisateur")]
    public partial class utilisateur
    {
        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2214:DoNotCallOverridableMethodsInConstructors")]
        public utilisateur()
        {
            animals = new HashSet<animal>();
            commandes = new HashSet<commande>();
        }

        public int ID { get; set; }

        [StringLength(32)]
        public string NOMCLI { get; set; }

        [StringLength(32)]
        public string PRENOMCLI { get; set; }

        [StringLength(50)]
        public string MAILCLI { get; set; }

        [StringLength(32)]
        public string PASSCLI { get; set; }

        [StringLength(32)]
        public string ADRCLI { get; set; }

        public int? CPCLI { get; set; }

        [StringLength(32)]
        public string VILLECLI { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<animal> animals { get; set; }

        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA2227:CollectionPropertiesShouldBeReadOnly")]
        public virtual ICollection<commande> commandes { get; set; }
    }
}
